packages/extension/src/view/devtools/stateProviders/filterManagementStore/utils/tests/filterCookiesWithRetentionPeriod.ts

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies.
+ */
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies.
+ */
+import type { CookieTableData } from '../../../../cookies.types';
+import filterCookiesWithRetentionPeriod from '../filterCookiesWithRetentionPeriod';
+
+export const frameFilteredCookies: { [key: string]: CookieTableData } = {
+  LSOLH: {
```

Comment:
I think this const can be moved to a different file as a testing util (./packages/extension/test-utils/consts.ts).

Reply:
Fixed in [99cf5a2](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/108/commits/99cf5a28ae719eb15c843ce6b4a39977f2cb1794)